src/Drive.php

```diff
@@ -17,12 +22,12 @@ class Drive
     private string $webDavUrl = '';
 
     /**
-     * @phpstan-var array{'headers'?:array<string, mixed>, 'verify'?:bool}
+     * @phpstan-var array{'headers'?:array<string, mixed>, 'verify'?:bool, 'webfinger'?:bool, 'guzzle'?:Client}
```

Comment:
```suggestion
     * @phpstan-var array{'headers'?:array<string, mixed>, 'verify'?:bool, 'webfinger'?:bool, 'guzzle'?:GuzzleHttp\Client}
```

Maybe you can write the full thing here, and not have to add `use GuzzleHttp\Client;` above?